vmdb/app/models/miq_provision_request.rb

```diff
@@ -120,4 +120,11 @@ def vms
   def originating_controller
     "vm"
   end
+
+  def template_valid?
```

Comment:
I REALLY dislike this pattern.  The convention is that methods ending in ? return something "truthy" or boolean.  This one is returning true or a string.  So, a caller cannot really use it naturally such as `if x.template_valid?`.